### DISCUSSION TITLE
FFD Points and Box Visualization

### DISCUSSION THREAD

**Discussion by usmanshahid90** [mdolab/dafoam#389] [General]
Hi,
I understand how to generate FFD points and box when the mesh is being made using genairfoilmesh.py .  But how to visualize the FFD box and points in paraview when the mesh is imported from some other software , e.g ICEM, how to visualize the FFD points then?

Looking forward to guidance from you.

Thanks.

**Comment by friedenhe** [comment]
Check FAQ.

**Reply by usmanshahid90** [reply]
Hi, I am running this command "dafoam_plot3d2tecplot.py yourFFDFileName.xyz newFFDFileName.dat. 
It is giving an error "dafoam_plot3d2tecplot.py" command not found. 
Kindly guide. 

Thanks
